### PR TITLE
[Reviewed] [Gamepad] Add behaviors for platformer characters and top-down movement

### PR DESCRIPTION
**Pull request by D8H**
### Examples
- https://github.com/GDevelopApp/GDevelop-examples/pull/415
- https://github.com/GDevelopApp/GDevelop-examples/pull/414

### CODE REVIEW

**Comment by Bouh** [comment]
The list of inputs is limited, we can't put the triggers LT BR RT RB. You should put all the keys. 
image

**Comment by Bouh** [comment]
I was in the Platformer behavior, and I saw this What is the propertie here?
I use the webapp, maybe it's not a version that is not update to date for testing?
image
image


I recommend to use groups, because what mean Use left stick I don't know what it refer, the group name helps:
image

**Comment by D8H** [comment]
I started from the wrong version 😭

File: extensions/reviewed/Gamepads.json [hunk 1]
```diff
@@ -1684,7 +1684,7 @@
                           "parameters": [
                             "",
                             "Object.Behavior::PropertyGamepadIdentifier()",
-                            "\"LEFT\"",
+                            "\"Left\"",
```
Comment (Bouh) [inline extensions/reviewed/Gamepads.json:1687]: There is two Left, **Right** is missing?

**Review by Bouh — APPROVED** [review]

**Comment by D8H** [comment]
I tested the last commit on both example.

**Pull request merged**